packages/calcite-components/src/components/popover/popover.scss

```diff
@@ -58,6 +58,13 @@
   border-radius: var(--calcite-popover-corner-radius, var(--calcite-corner-radius-round));
 }
 
+.calcite-floating-ui-arrow {
+  fill: var(--calcite-popover-background-color, var(--calcite-color-foreground-1));
+  .calcite-floating-ui-arrow__stroke {
```

Comment:
Does this need the extra specificity from nesting or can the stroke styles be moved outside?

Reply:
No it doesn't, I moved it outside.